sentry-android-core/src/test/java/io/sentry/android/core/AppStartStateTest.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.core
+
+import java.util.Date
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertNull
+import kotlin.test.assertSame
+
+class AppStartStateTest {
```

Comment:
`m`: I think we are still missing a test to validate the calculation of `getAppStartInterval`.

Reply:
good point